optuna/integration/cma.py

```diff
@@ -183,7 +183,7 @@ def sample_independent(
         self._raise_error_if_multi_objective(study)
 
         if self._warn_independent_sampling:
-            complete_trials = [t for t in study.trials if t.state == TrialState.COMPLETE]
+            complete_trials = study.get_trials(deepcopy=False, states=(TrialState.COMPLETE,))
```

Comment:
`complete_trials` is necessary to count the number of complete trials, so we can use `deepcopy=False`.